sbin/common/config_init.sh

```diff
@@ -31,6 +31,7 @@
 CONFIG_PARAMS=(
 ADOPTIUM_DEVKIT_LOCATION
 ADOPT_PATCHES
+ALSA
```

Comment:
Hmm, is this label to indicate that we skip Alsa or something else?

Reply:
I got inspired by `FREETYPE`  whic have the same name style and is doinf practically the same.